src/style.css

```diff
@@ -1,96 +1,383 @@
+  bottom: 3px;
+  background-color: #fff;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+input:checked + .slider {
+  background-color: var(--font-color);
 }
-.logo.vanilla:hover {
-  filter: drop-shadow(0 0 2em #3178c6aa);
+
+input:focus + .slider {
+  box-shadow: 0 0 1px var(--font-color);
 }
 
-.card {
-  padding: 2em;
+input:checked + .slider:before {
+  -webkit-transform: translateX(1.8rem);
+  -ms-transform: translateX(1.8rem);
+  transform: translateX(1.8rem);
 }
 
-.read-the-docs {
-  color: #888;
+/* Rounded sliders */
+.slider.round {
+  border-radius: 2rem;
 }
 
-button {
-  border-radius: 8px;
-  border: 1px solid transparent;
-  padding: 0.6em 1.2em;
-  font-size: 1em;
+.slider.round:before {
+  border-radius: 50%;
+}
+
+/*
+-----------------------------------
+Search-box
+-----------------------------------
+*/
+.search_box_container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.inner_search_box {
+  border: 1px solid var(--border-line-color);
+  box-shadow: 0 3px 12px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.08);
+  display: flex;
+  align-items: center;
+  border-radius: var(--border-radius-main);
+  font-size: 0.75rem;
   font-weight: 500;
-  font-family: inherit;
-  background-color: #1a1a1a;
-  cursor: pointer;
-  transition: border-color 0.25s;
-}
-button:hover {
-  border-color: #646cff;
-}
-button:focus,
-button:focus-visible {
-  outline: 4px auto -webkit-focus-ring-color;
-}
-
-@media (prefers-color-scheme: light) {
-  :root {
-    color: #213547;
-    background-color: #ffffff;
-  }
-  a:hover {
-    color: #747bff;
-  }
-  button {
-    background-color: #f9f9f9;
-  }
+}
+/* 
+-----------------------------
+hover searchbox
+-----------------------------
+*/
+.where_section {
+  padding: 1rem 7rem 1rem 2rem;
+  position: relative;
+}
+
+.active_section {
+  box-shadow: 0 3px 12px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.08);
+  background-color: #fff;
+  border-radius: var(--border-radius-main);
+}
+.bg-active .active_box:hover {
+  background-color: var(--border-line-color);
+}
+.bg-active .active_section:hover {
+  background-color: #fff;
+}
+.active_box:hover {
+  border-radius: var(--border-radius-main);
+  background-color: var(--background-hover);
+}
+.active_box:hover::after {
+  opacity: 0;
+}
+.bg-active {
+  background-color: var(--background-hover);
+}
+.bg-active .search_btn button {
+  width: 7rem;
+  /* border-radius: 500px; */
+}
+.bg-active .search_content {
+  display: inline-block;
+  padding-left: 0.2rem;
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+/*
+ ---------------------------------------
+ */
+/* .where_section::after,
+.checkout_section::after,
+.checkin_section::after {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 50%;
+  transform: translateY(50%);
+  border-right: 1px solid var(--border-line-color);
+  opacity: 1;
+} */
+.divider {
```

Comment:
It is better to use more specific class names. For example, "divider" is very common, you may later forget it and put it for another item or even your teammate may put it for another part. You can use something like "header-divider" here.